### PR TITLE
[Fixed] DUO Admin - Auth logs bug

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19141

## Description
The usage of the api in the command was wrong. The param passed to the API has to be users and not username.
By default if no users are specified then the response returned is auth logs for all users, which led to same results for different users.

## Bug Fix Testing
The PB have been enhanced to check for the bug fix.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Code Review
- [x] Technical Writer Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/2d301fa941d791fcfccd6ebb17a0042279d9bff8/Integrations/DuoAdminApi/DuoAdminApi.yml) - line 62 - limit parameter description
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/2d301fa941d791fcfccd6ebb17a0042279d9bff8/Integrations/DuoAdminApi/CHANGELOG.md) - line 2 - needs to be checked.

### CODE REVIEW

File: Integrations/DuoAdminApi/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+-
```
Comment (Itay4) [inline Integrations/DuoAdminApi/CHANGELOG.md]: add release notes

File: Integrations/DuoAdminApi/DuoAdminApi.py [hunk 2]
```diff
@@ -146,7 +146,7 @@ def get_all_users():
 def get_authentication_logs_by_user(username, mintime):
     res = admin_api.get_authentication_log(
         2,
-        username=username,
+        users=get_user_id(username),
         mintime=time_to_timestamp_milliseconds(OPTIONS_TO_TIME[mintime]),
         maxtime=time_to_timestamp_milliseconds(datetime.now()),
         limit='50'
```
Comment (Itay4) [inline Integrations/DuoAdminApi/DuoAdminApi.py]: make this an argument

**Comment by guyfreund** [comment]
@kirbles19 please notice the review details in the issue description.

**Comment by guyfreund** [comment]
@Itay4 - PB have been updated. If running the PB with the updated code of the integration the PB succeeds, and if running with the old code the PB fails.

**Comment by kirbles19** [comment]
@guyfreund @Itay4 Done with the minor fixes listed, but this entire integration needs to be reviewed. The docs-approved only applies to the single line updated.

File: Integrations/DuoAdminApi/DuoAdminApi.yml [hunk 3]
```diff
@@ -56,6 +56,13 @@ script:
       - 10_years_ago
       required: true
       secret: false
+    - default: false
+      defaultValue: '50'
+      description: The maximum numner of authentication logs to return.
```
Comment (Itay4) [inline Integrations/DuoAdminApi/DuoAdminApi.yml]:
```suggestion
      description: The maximum number of authentication logs to return.
```

File: Integrations/DuoAdminApi/DuoAdminApi.py [hunk 4]
```diff
@@ -144,12 +144,13 @@ def get_all_users():
 
 
 def get_authentication_logs_by_user(username, mintime):
+    limit = demisto.args().get('limit')
```
Comment (Itay4) [inline Integrations/DuoAdminApi/DuoAdminApi.py]:
```suggestion
    limit = demisto.args().get('limit', '50')
```

File: TestPlaybooks/playbook-DuoAdminaAPITest.yml [hunk 5]
```diff
@@ -1,14 +1,14 @@
 id: DuoAdmin API test playbook
-version: -1
+version: 22
```
Comment (Itay4) [inline TestPlaybooks/playbook-DuoAdminaAPITest.yml]:
```suggestion
version: -1
```

File: TestPlaybooks/playbook-DuoAdminaAPITest.yml [hunk 6]
```diff
@@ -84,17 +87,21 @@ tasks:
...
     id: "9"
-    taskid: 109cd4fe-d13f-4b06-88e6-3585a8039713
+    taskid: 61d2af40-0f4e-4179-850d-5f1fdd026d96
     type: title
     task:
-      id: 109cd4fe-d13f-4b06-88e6-3585a8039713
+      id: 61d2af40-0f4e-4179-850d-5f1fdd026d96
       version: -1
-      name: Done
+      name: Bug Fix Test
```
Comment (Itay4) [inline TestPlaybooks/playbook-DuoAdminaAPITest.yml]: ?

File: TestPlaybooks/playbook-DuoAdminaAPITest.yml [hunk 7]
```diff
@@ -796,12 +822,145 @@ tasks:
...
+    timertriggers: []
+    ignoreworker: false
+  "45":
+    id: "45"
+    taskid: 6d2ab6bd-cf01-47b6-87ba-a640566d8e58
+    type: condition
+    task:
+      id: 6d2ab6bd-cf01-47b6-87ba-a640566d8e58
+      version: -1
+      name: Check if User0 doesn't exists and User2 does exsits
```
Comment (Itay4) [inline TestPlaybooks/playbook-DuoAdminaAPITest.yml]:
```suggestion
      name: Check if User0 doesn't exists and User2 does exist
```

File: TestPlaybooks/playbook-DuoAdminaAPITest.yml (hunk 6 above)
Comment (Itay4) [reply TestPlaybooks/playbook-DuoAdminaAPITest.yml]: ?

**Pull request merged**